### ISSUE TITLE
Inquiry regarding the OmniGen2's reinforcement learning aspects

### ISSUE THREAD

**Issue by ODRLver** [VectorSpaceLab/OmniGen2#127]
Hi! First off, I want to express my sincere appreciation for the incredible work!
I've recently been working on applying FLOW-GRPO into OmniGen2 to test if the reinforcement learning could improve the model's output quality. However, after some hyperparameters adjusting and code debugging, I couldn't manage to raise the average reward. I figured your team might have nailed this kind of post-training process, I would really appreciate any help you can spare, on how to modify my code or maybe hyperparameters.
The average reward curve looks like this:

<img width="795" height="300" alt="Image" src="https://github.com/user-attachments/assets/6a958083-28e6-4c26-8a8a-35d012b351ef" />

During training, my denoising steps are 10 steps, and the noise level is set between 0.2-0.3. I've also noticed that converting from ODE to SDE could cause the model to generate a lot of colorful dots on the image, looks like this:(from top to below, the training goes on).
Even adjusting the noise level to a relatively low level could also cause it sometimes.

<img width="519" height="540" alt="Image" src="https://github.com/user-attachments/assets/b1328912-6f58-4efc-96b7-49b0cd4e1733" />

<img width="519" height="540" alt="Image" src="https://github.com/user-attachments/assets/d6587640-c7d5-49c9-b00f-e332d621458b" />

<img width="519" height="540" alt="Image" src="https://github.com/user-attachments/assets/a80cecf8-d16a-4bb7-8ae7-e0329e0d9ace" />

<img width="519" height="540" alt="Image" src="https://github.com/user-attachments/assets/bbd0241c-13c5-45b1-8fa1-e19e20a4c4ad" />

**Comment by hrz2000** [comment]
hi @ODRLver 

omnigen2 scheduler的timestep是从0到1，sigma(_timesteps)也是从0到1。不太兼容flow-grpo的sde step（它的sigma是从1到0），简单的取反，生成的结果噪声很多，请问您解决这个问题了吗，期待您的回复讨论！

**Comment by Luciennnnnnn** [comment]
Hi, the RL version has been updated, see [news](https://github.com/VectorSpaceLab/OmniGen2?tab=readme-ov-file#-news), powered by our new [reward model](https://github.com/VectorSpaceLab/EditScore).

**Comment by ODRLver** [comment]
> Hi [@hrz2000](https://github.com/hrz2000) [@ODRLver](https://github.com/ODRLver) , the SDE of OmniGen2 differs due to its distinct time schedule. We provide a detailed derivation in the appendix of our new [paper](https://arxiv.org/abs/2506.18871). In addition, we will release the RL training code for OmniGen2, which supports dynamic resolution and variable numbers of images per batch.

Nice work, elegant mathematical derivation! I believe the paper you mention here is actually this one: [paper](https://arxiv.org/pdf/2509.23909)

**Comment by Luciennnnnnn** [comment]
@ODRLver You are correct! Thanks for the correction.
